.github/workflows/main.yml

```diff
@@ -0,0 +1,62 @@
+name: ROHQ Build/Release
+
+on:
+  push:
```

Comment:
The [build file inside MyLA repository,](https://github.com/tl-its-umich-edu/my-learning-analytics/blob/master/.github/workflows/main.yml) include tag push. Do we need to include the tags here for ROHQ?

Reply:
I removed them because I don't _think_ the `on: push: tags:` is actually necessary for having this work. We would manually trigger a release for that tag and don't need it to automatically trigger. It created a tagged package as expected for my. I don't think it's needed for MyLA either and just duplicates the manual release. We should never be pushing changes to a tag. 

If it _was_ needed it would require a little research as the tag format currently for ROHQ is different and uses semver rather than calver.

Reply:
Make sense. No need to include the tags trigger.